src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/JavaASTAnalyser.java

```diff
@@ -615,9 +617,10 @@ private void getEnumEntries(EnumDeclaration enumDeclaration) {
                 // annotations
                 getAnnotations(enumConstantDeclaration, false, false);
 
-                // create a unique id for enum constants
+                // create a unique id for enum constants by using the fully-qualified constant name
+                // (package, enum name, and enum constant name)
                 final String name = enumConstantDeclaration.getNameAsString();
-                final String definitionId = makeId(enumDeclaration.getFullyQualifiedName().get() + "." + counter);
```

Comment:
I believe this will cause to create a diff and a new revision though nothing is different in UI because diff comparison compares both text and underlying ID. One option to avoid that issue is to increase the version of parser to rebuild all existing revisions.